test/transcoding/fpga_lsu_optimized.ll

```diff
@@ -0,0 +1,166 @@
+; LLVM IR generated by Intel SYCL Clang compiler (https://github.com/intel/llvm)
+; SYCL source code can be found below:
+
+; #include <CL/sycl.hpp>
+; #include <CL/sycl/intel/fpga_extensions.hpp>
+;
+; int test_lsu(cl::sycl::queue Queue) {
+;   int output_data[2];
+;   for (size_t i = 0; i < 2; i++) {
+;     output_data[i] = -1;
+;   }
+;
+;   int input_data[2];
+;   for (size_t i = 0; i < 2; i++) {
+;     input_data[i] = i + 1;
+;   }
+;
```

Comment:
```suggestion
```

I think there is no need to put host code in there as it anyway is not emitted to LLVM IR. The same applies to code that validates results after kernel enqueue

Reply:
Thank you for a quick feedback. Removed.